### DISCUSSION TITLE
Cloudflared 2023.4.0 is released

### DISCUSSION THREAD

**Discussion by Delta1977** [homeassistant-apps/app-cloudflared#371] [Ideas]
Hi, cloudflared 2023.4.0 is released. Can you updated your addon ?

**Comment by addvanced** [comment]
As far as I can tell, renovatebot already merged it into the main branch.
So all we need is a new release!

So, would it be possible for you @brenner-tobias to create a new release with the updated Cloudflared service?

Thanks!

**Comment by brenner-tobias** [comment]
Please give us some time, we do update the Edge Repository automatically immediately but do check if there were any breaking changes with Cloudflared and this add-on or new issues arise, so I usually wait a couple of days and check for issues in the Cloudflared Add-On and our Repo / Edge Repo.

So if you want to have the new Cloudflared Version as soon as it is released, feel free to switch to the [Edge Repo](https://github.com/brenner-tobias/ha-addons-edge). Otherwise you have to wait for a couple of days.
